shaded-ning19/pom.xml

```diff
@@ -23,13 +23,14 @@
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
   <modelVersion>4.0.0</modelVersion>
   <parent>
-    <groupId>org.apache.apex</groupId>
-    <artifactId>apex</artifactId>
-    <version>3.3.0-incubating</version>
+    <groupId>org.apache</groupId>
+    <artifactId>apache</artifactId>
+    <version>18</version>
+    <relativePath/>
   </parent>
   <groupId>org.apache.apex</groupId>
   <artifactId>apex-shaded-ning19</artifactId>
-  <version>1.0.0</version>
+  <version>1.1.0-SNAPSHOT</version>
```

Comment:
The same question as for codestyle-config. Do we want to release Apex Core with SNAPSHOT version? If not, will it be necessary to release shaded-ning19 before the next Apex Core release?

codestyle-config and shaded ning may technically reside in a different git repository and not be part of the Apex core source release. IMO, it will be better to avoid any dependency of codestyle-config and shaded ning on the Apex core including commits.